tests/vmi_networking_test.go

```diff
@@ -594,6 +594,70 @@ var _ = Describe("Networking", func() {
 		})
 	})
 
+	Context("VirtualMachineInstance with masquerade binding mechanism", func() {
+		var serverVMI *v1.VirtualMachineInstance
+		var clientVMI *v1.VirtualMachineInstance
+
+		It("should allow regular network connection", func() {
+			By("creating two virtual machines")
+			ports := []v1.Port{{Name: "http", Port: 8080}}
+			serverVMI = tests.NewRandomVMIWithMasqueradeInterfaceEphemeralDiskAndUserdata(tests.ContainerDiskFor(tests.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n", ports)
```

Comment:
that's quite a long func name `NewRandomVMIWithMasqueradeInterfaceEphemeralDiskAndUserdata` :), may I suggest to rename it and move it to local `Context` scope? You're using this function only here so I have doubts if we need it in `utils`. My opinion is, if someone will need to use it somewhere else than he/she should moved it then.